modules/distribution/src/repository/jaggeryapps/cdm/modules/constants.js

```diff
@@ -16,6 +16,7 @@
  * under the License.
  */
 var WEB_APP_TITLE = "WSO2 MDM";
+var WEB_APP_CONTEXT = "/cdm/";
```

Comment:
The context of the app is cdm. This is because I am creating another app with the context of mdm that has the codebase of the new unified store based one.